src/basic/parse-util.h

```diff
@@ -11,6 +11,32 @@
 
 typedef unsigned long loadavg_t;
 
+#define TPM2_PCRS_MAX 24U
+typedef enum PcrIndex {
+/* The following names for PCRs 0…7 are based on the names in the "TCG PC Client Specific Platform Firmware Profile Specification" (https://trustedcomputinggroup.org/resource/pc-client-specific-platform-firmware-profile-specification/) */
+        PCR_PLATFORM_CODE       = 0,
+        PCR_PLATFORM_CONFIG     = 1,
+        PCR_EXTERNAL_CODE       = 2,
+        PCR_EXTERNAL_CONFIG     = 3,
+        PCR_BOOT_LOADER_CODE    = 4,
+        PCR_BOOT_LOADER_CONFIG  = 5,
+        PCR_SECURE_BOOT_POLICY  = 7,
+/* The following names for PCRs 9…15 are based on the "Linux TPM PCR Registry"
+(https://uapi-group.org/specifications/specs/linux_tpm_pcr_registry/) */
+        PCR_KERNEL_INITRD       = 9,
+        PCR_IMA                 = 10,
+        PCR_KERNEL_BOOT         = 11,
+        PCR_KERNEL_CONFIG       = 12,
+        PCR_SYSEXTS             = 13,
+        PCR_SHIM_POLICY         = 14,
+        PCR_SYSTEM_IDENTITY     = 15,
+/* As per "TCG PC Client Specific Platform Firmware Profile Specification" again, see above */
+        PCR_DEBUG               = 16,
+        PCR_APPLICATION_SUPPORT = 23,
+        _PCR_INDEX_MAX_DEFINED  = TPM2_PCRS_MAX,
+        _PCR_INDEX_INVALID      = -EINVAL,
+} PcrIndex;
```

Comment:
hmm, this shouldn't be defined in parse-util.h, which contains pretty generic parsers.

stuff specific to tpm should really be in tpm2-util.h.

Reply:
> hmm, this shouldn't be defined in parse-util.h, which contains pretty generic parsers.
> 
> stuff specific to tpm should really be in tpm2-util.h.

Awesome. Thanks for the guidance, you guys help me get better always. Thank you!